.github/workflows/build_and_test.yml

```diff
@@ -70,6 +70,18 @@ jobs:
       with:
         name: artefacts
         path: wheelhouse/
+    - name: Install docs dependencies
+      if:  (matrix.os == 'ubuntu-20.04') && (github.event_name == 'pull_request')
+      run: |
+        pip install -r .github/workflows/docs/requirements.txt
+    - name: Build docs
+      if:  (matrix.os == 'ubuntu-20.04') && (github.event_name == 'pull_request')
+      timeout-minutes: 20
+      run: |
+        cd .github/workflows/docs
+        mkdir extensions
+        ./build-docs -a -i -d ${GITHUB_WORKSPACE}/.github/workflows/docs/extensions
```

Comment:
We should maybe check the output for warnings, as we do for the pytket docs. I see this in the logfile:
```
2022-08-04T09:42:19.4321218Z /opt/hostedtoolcache/Python/3.10.5/x64/lib/python3.10/site-packages/pytket/extensions/quantinuum/backends/quantinuum.py:docstring of pytket.extensions.quantinuum.backends.quantinuum.QuantinuumBackend.add_to_batch:3: WARNING: Unexpected indentation.
```

Reply:
done